aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -110,13 +111,20 @@ static void check_from_to_in_range(int64_t from, int64_t to_inc, caffe2::TypeMet
       WARN_OUT_OF_BOUNDS(from, "from", digits, dtype);
       WARN_OUT_OF_BOUNDS(to_inc, "to - 1", digits, dtype);
     });
+  } else if (scalar_type == kUInt64) {
+    AT_DISPATCH_V2(scalar_type, "check_random_integral_bounds", AT_WRAP([&]() {
```

Comment:
What is the benefit of the dispatch macro if you only give it a single dtype? Is it not a no-op?

Reply:
Hmm, actually I guess this is kind of pointless, especially here where I'm not using `scalar_t` lol.